config/routes.rb

```diff
@@ -1,5 +1,8 @@
 MANIACChess::Application.routes.draw do
   devise_for :users
+
+  resources :games
```

Comment:
You're creating a number of routes here that aren't supported by the controller. Could you narrow this down using `only` or `except`?